lib/libmpa/include/mpa_debug.h

```diff
@@ -58,19 +58,19 @@ void __mpa_dbg_dump_mpanum(const mpanum val);
 
 #define DPRINT   \
 	DPRINT_WHERE \
-	__mpa_dbg_print
+	DMSG_RAW
```

Comment:
This is inconsistent. `DPRINT_WHERE` calls `__mpa_dbg_print_header()` which does `MSG_RAW(...)`. As a result, the file/line information is printed whenever `TRACE_LEVEL > 0` but the rest of the message is shown only when `TRACE_LEVEL >= TRACE_DEBUG`.
I see two options: either keep `__mpa_dbg_print()`, or replace `MSG_RAW()` with `DMSG_RAW()` in all `__mpa_dbg_{print,dump}*()` functions.